interface/src/app/node/node.js

```diff
@@ -5,7 +5,10 @@
 angular.module('adage.node', [
   'ui.router',
   'ui.bootstrap',
-  'ngResource'
+  'ngResource',
+  // Dependency modules from the 'adage' Angular app
+  'adage.utils'
```

Comment:
Ok, I actually really like the idea of having a 'greenelab.stats' module (inside a greenelab folder), so I switched it to that. That way, we can just make everything in the 'greenelab' folder be portable Angular modules that we can use in different servers.